Pipeline/tests/ModuleInfoRetrievalTests.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+import json
+import os
+from .constants.TestPayloads import *
+from .constants.TestEnvironment import *
+from .utils.MockResponse import *
+# relative paths refuse to work because Python is terrible!
+from app import app
+from mock import patch, Mock
+import io
+
+CURRENT_DIRECTORY = os.path.dirname(__file__)
+
+# NOTE: Results here can be considered determnistic but note that dataset changes mean that the responses also need to be changed.
+# NOTE: For boolean based comparisons, we still use assertEquals so there is no need to update code in the case of a new response (i.e. you only need to change the expected response JSONs)
+
+
+class ModuleInfoRetrievalTests(unittest.TestCase):
```

Comment:
is this endpoint ever called in combination with vernal or rnamigos? cuz if yes it would be nice to have tests in combo with those calls. idk if that would be redundant though if they're already tested enough elsewhere...

Reply:
no it's completely independent